From f5d91cb16fcf7b82f43816cc5f4264fb4103c843 Mon Sep 17 00:00:00 2001
From: "Vladimir S. FONOV" <vladimir.fonov@gmail.com>
Date: Tue, 4 Feb 2014 16:49:06 -0500
Subject: [PATCH] Fixed use of uninitialized variable

---
 progs/mincresample/resample_volumes.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/progs/mincresample/resample_volumes.c b/progs/mincresample/resample_volumes.c
index 841f8a3..655134a 100644
--- a/progs/mincresample/resample_volumes.c
+++ b/progs/mincresample/resample_volumes.c
@@ -529,11 +529,11 @@ void get_slice(long slice_num, VVolume *in_vol, VVolume *out_vol,
       to get an appropriate error margin (ftol) for the function
       grid_inverse_transform_point */
    for (idim_in=0; idim_in < in_vol->file->ndims; idim_in++) {
-           
+      
       /* Get size of dimension */
       (void) ncdiminq(in_vol->file->mincid, dim[idim_in], dimname, 
-                      &in_vol->file->nelements[idim]);
-
+               &in_vol->file->nelements[idim_in]);
+      
       /* Check for existence of variable */
       dimid = ncvarid(in_vol->file->mincid, dimname);
       if (dimid == MI_ERROR) continue;
